configure.ac

```diff
@@ -1524,7 +1524,7 @@ AM_CONDITIONAL([ENABLE_SYSCALL_SANDBOX], [test "$use_syscall_sandbox" != "no"])
 
 dnl Check for reduced exports
 if test "$use_reduce_exports" = "yes"; then
-  AX_CHECK_COMPILE_FLAG([-fvisibility=hidden], [CORE_CXXFLAGS="$CORE_CXXFLAGS -fvisibility=hidden"],
+  AX_CHECK_COMPILE_FLAG([-fvisibility=hidden], [RE_CXXFLAGS="$RE_CXXFLAGS -fvisibility=hidden"],
```

Comment:
In commit "build: Put reduced exports flags in RE_CXXFLAGS" (bd1376c178df10cf64faa8d2e810149ff0ff0d49)

The commit message rationale about "we really shouldn't be setting CXXFLAGS" doesn't seem to apply anymore now that current code is setting CORE_CXXFLAGS. But maybe another reason for changing this is to make R_CXXFLAGS consististent with RE_LDFLAGS?

Commit message should probably be fixed in any case

Reply:
I removed the `REFLAGS` changes from this PR, thanks for the suggestion!